pyvista/plotting/renderer.py

```diff
@@ -437,48 +439,50 @@ def disable_depth_peeling(self):
         self.SetUseDepthPeeling(False)
         self.Modified()
 
-    def enable_anti_aliasing(self):
-        """Enable anti-aliasing using FXAA.
+    def enable_anti_aliasing(self, aa_type):
```

Comment:
Yes. Sorry about that.

With these big changes I'm considering adding in some optional workflows to verify we work against `mne`.

Reply:
> I'm a CI ignoramus but it would make sense to me to have a dedicated CI run to test downstream. At the moment this would only include MNE, but it's plausible that we'd want to test other compatibility things against downstream, and then we could add that there.

Absolutely, and since their workflow in #3255 takes less time than our longest run, I'd really like for us to test against multiple downstream packages.